Topic 0/src/Classes/Builder.java

```diff
@@ -0,0 +1,56 @@
+package Classes;
+
+public abstract class Builder {
+	public class Home {
+
+	}
+
+	private int rooms;
+	private int windows;
+	private String street;
+	private String phone;
+	
+	public Builder(int r, int w, String s, String p) {
```

Comment:
Please use proper parameter names, that you can understand what they mean. For example,
  <code>public Builder (int roomNumber, int windowNumber, String street, String phoneNumber)</code>